x/ibc-rate-limit/contracts/rate-limiter/src/execute.rs

```diff
@@ -156,23 +156,23 @@ mod tests {
     };
     use crate::ContractError;
 
-    const IBC_ADDR: &str = "osmo1vz5e6tzdjlzy2f7pjvx0ecv96h8r4m2y92thdm";
-    const GOV_ADDR: &str = "osmo1tzz5zf2u68t00un2j4lrrnkt2ztd46kfzfp58r";
+    const IBC_ADDR: &str = "neutron1cdlz8scnf3mmxdnf4njmtp7vz4gps7fswphrqn";
+    const GOV_ADDR: &str = "neutron1w02khza7ux68ccwmz2hln97mkjspjxes8y2k9v";
 
     #[test] // Tests AddPath and RemovePath messages
     fn management_add_and_remove_path() {
         let mut deps = mock_dependencies();
         IBCMODULE
-            .save(deps.as_mut().storage, &Addr::unchecked(IBC_ADDR))
+            .save(deps.as_mut().storage, &MockApi::default().addr_make(IBC_ADDR))
```

Comment:
why do you prefer `&MockApi::default().addr_make` over `Addr::unchecked`?

Reply:
After upgrading to `2.x` of `cosmwasm-std` unit tests were triggering the following error

```
called `Result::unwrap()` on an `Err` value: Std(GenericErr { msg: "Error decoding bech32", backtrace: <disabled> })
```

When reading through the [migration](https://github.com/CosmWasm/cosmwasm/blob/main/MIGRATING.md) docs for upgrading to `2.x` this is mentioned

> Replace addresses in unit tests with valid bech32 addresses. This has to be done for all addresses that are validated or canonicalized during the test or within the contract. The easiest way to do this is by using MockApi::addr_make. It generates a bech32 address from any string:

So I just ended up replacing all instances of `Addr::unchecked` with `MockApi::default().addr_make`